docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -203,6 +203,27 @@ spec:
           configDrive: true
 ```
 
+If you are using Kuryr and your SR-IOV network has `port_security_enabled=False`, then
+you'll be forced to create the worker node setting its primary port with `portSecurity: false`
```

Comment:
"You'll be forced" or "you must"?

Reply:
Docs typically prefers "must" in contexts like that.

Reply:
Alright!